packages/web-app-files/src/index.js

```diff
@@ -23,65 +35,72 @@ const appInfo = {
 }
 const navItems = [
   {
-    name: $gettext('All files'),
+    name: $gettext(`All files`),
```

Comment:
What motivated this change here?

Reply:
good catch, i had dynamic variables in there while refactoring. I removed it